hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SaslPropertiesResolver.java

```diff
@@ -23,6 +23,7 @@
 
 import javax.security.sasl.Sasl;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
I'm 99% sure that upstream is using shaded guava here. Better if you could use a data structure from the standard library.